www/js/main.js

```diff
@@ -96,8 +96,29 @@ angular.module('emission.main', ['emission.main.recent', 'emission.main.diary',
     }
 })
 
-.controller('ControlCtrl', function($scope, $window, $ionicScrollDelegate, $state, $ionicPopup, $ionicActionSheet, $ionicPopover, $rootScope) {
+.controller('ControlCtrl', function($scope, $window, $ionicScrollDelegate, $state, $ionicPopup, $ionicActionSheet, $ionicPopover, $rootScope, ControlHelper) {
+    $scope.emailLog = ControlHelper.emailLog;
     $scope.dark_theme = $rootScope.dark_theme;
+
+    
+    $scope.toggleLowAccuracy = function() {
+        if ($scope.settings.collect.config == null) {
+            return false; // config not loaded when loading ui, set default as false
```

Comment:
Here's where the screen (used to be?) refreshed
https://github.com/e-mission/e-mission-phone/blob/master/www/js/main.js#L384